src/widgets/scrollbar.rs

```diff
@@ -117,19 +137,33 @@ impl ScrollbarState {
     }
 }
 
-/// Scrollbar Orientation
+/// This is the position of the scrollbar around a given area.
+///
+/// ```plain
+///           HorizontalTop
+///             ┌───────┐
+/// VerticalLeft│       │VerticalRight
+///             └───────┘
+///          HorizontalBottom
+/// ```
```

Comment:
I think scrollbar orientation should be:

- `Horizontal`
- `Vertical`

With another `ScrollbarPosition` as `Start` and `End`.

Maybe leave this undocumented for now?

Reply:
For me scrollbar _orientation_ isn't even a good name. There is definitely room for improvement here. However, in the meantime those improvement are being thought/worked on I'm for keeping the documentation so that users and future maintainers know what this is.

Reply:
Makes sense! We should open an issue to track that discussion.